### DISCUSSION TITLE
Waters RAW data conversion - error string to float

### DISCUSSION THREAD

**Discussion by brvpuyve** [michaelmarty/UniDec#98] [Q&A] [answered]
Hi UniDec team,

We are trying to convert our Waters .RAW data using the UniDec Data Conversion Wizard, however it does not seem to work because of a string to float issue '(eV)'. Is there an easy way to fix this issue?

As example, I hereby provide you with a download link to one of our .RAW files of Myoglobin:

https://filesender.belnet.be/?s=download&token=8a3f41aa-4e24-4906-ab93-903c49dcfd69 

Best,
Bart Van Puyvelde
Ghent University

**Comment by michaelmarty** [comment, marked as answer]
Hi Bart,
Thanks for sending that and for letting me know. Waters has two lines now for "Trap Collision Energy" and "Trap Collision Energy (eV)". My test files didn't have that, so thanks for sending yours along. The new version should have that fixed. 

Can you try this and let me know how it works?: https://www.dropbox.com/scl/fi/x691mwkn14abd758flpql/UniDec_Windows_240626.zip?rlkey=7wx7ydsw75uxc47zp0yisvyxa&dl=0

It's also committed on the v3 branch. Thanks!
MTM

**Reply by brvpuyve** [reply]
Hi Michael,

Thanks for the swift response! I just tried this version and now it works perfectly fine. 

Thank you very much!

Best,
Bart

**Comment by michaelmarty** [comment]
Awesome! Thanks again for helping track this down! Let me know if you have any future suggestions or issues. Thanks, MTM
